src/styles/blocks/products.scss

```diff
@@ -0,0 +1,19 @@
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  &__product-card {
+    display: flex;
+    margin-left: 48px;
+    margin-bottom: 50px;
+  }
+
+  &__product-card--p--is-first-in-row {
+    margin-left: 40px;
+  }
+  
+  &__product-card--p--is-last-in-row {
+    margin-right: 40px;
+  }
```

Comment:
It's not a good practice to hard-code such classes and margins. Let's make it in some other way.
As a variant - give max-width for your container `products` with `space-around`